listenbrainz/bigquery.py

```diff
@@ -1,10 +1,16 @@
+import googleapiclient
+import logging
 import os
+
 from googleapiclient import discovery
-import googleapiclient
 from oauth2client.client import GoogleCredentials
 
 APP_CREDENTIALS_FILE = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS')
 
+
+logger = logging.getLogger(__name__)
```

Comment:
You added logging to this module, but it is never used. What's your plan here?

Reply:
It was used in this file but I had forgotten to create the logger itself :( https://github.com/paramsingh/listenbrainz-server/blob/a1011edd21a5f330bd6617482baeb91664a39257/listenbrainz/bigquery.py#L18